Formula/geographiclib.rb

```diff
@@ -18,13 +19,11 @@ class Geographiclib < Formula
   depends_on "cmake" => :build
 
   def install
-    mkdir "build" do
-      args = std_cmake_args
-      args << "-DCMAKE_OSX_SYSROOT=#{MacOS.sdk_path}" if OS.mac?
-      args << "-DEXAMPLEDIR="
-      system "cmake", "..", *args
-      system "make", "install"
-    end
+    args = "-DEXAMPLEDIR="
```

Comment:
```suggestion
    args = ["-DEXAMPLEDIR="]
```
Otherwise the next line will concatenate the two strings.